include/drjit-core/jit.h

```diff
@@ -494,9 +494,14 @@ extern JIT_EXPORT void jit_registry_remove(const void *ptr);
 extern JIT_EXPORT uint32_t jit_registry_id(const void *ptr);
 
 /// Return the largest instance ID for the given domain
+/// If the \c domain is a nullptr, it returns the largest instance ID for the given backend
```

Comment:
Actually it looks like it just counts the number of *active* entries in a domain, not the largest instance ID?